crates/sui-framework/packages/sui-framework/sources/test/test_scenario.move

```diff
@@ -353,9 +365,36 @@ module sui::test_scenario {
         sui::transfer::share_object_impl(t)
     }
 
+    public fun get_objects_owned_by<T: key>(object_id: ID): vector<ID> {
+        ids_for_address<T>(object::id_to_address(&object_id))
+    }
+
+    public fun get_receiving_ticket_for_object<T: key>(parent: &ID): sui::transfer::Receiving<T> {
```

Comment:
I find this name a bit confusing, since I didn't immediately tell it was for the parent?

Reply:
It doesn't seem to line up with `get_receiving_ticket_for_object_by_id`